content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,124 @@
+
+We’re pleased to announce the release of Kubernetes 1.21, our first release of 2021!
+This release consists of 49 enhancements: 15 enhancements have graduated to stable, 15 enhancements are moving to beta, and 19 enhancements are entering alpha.
+
+## Major Themes
+
+### CronJobs Graduate to Stable!
+[CronJobs](/docs/concepts/workloads/controllers/cron-jobs/) (previously ScheduledJobs) has been a beta feature since Kubernetes 1.8! With 1.21 we get to finally see this widely used API graduate to stable.
+
+CronJobs are meant for performing all time-related actions, namely backups, report generation and so on. Each of these tasks should be allowed to run repeatedly (once a day/month, etc.) or once at a given point in time.
+
+### Immutable Secrets and ConfigMaps
+[Immutable Secrets](/docs/concepts/configuration/secret/#secret-immutable) and [ConfigMaps](/docs/concepts/configuration/configmap/#configmap-immutable) adds a new field to those resource types which will reject changes to those objects if set. Secrets and ConfigMaps by default are mutable which is beneficial for pods that are able to consume changes. Mutating Secrets and ConfigMaps can also cause problems if a bad configuration is pushed for pods that use them.
+
+By marking Secrets and ConfigMaps as immutable you can be sure your application configuration won't change. If you want to make changes you'll need to create a new, uniquly named Secret or ConfigMap and deploy a new pod to consume that resource. Immutable resources also have scaling benefits because controllers do not need to poll the API server to watch for changes.
+This feature is now graduating to stable in Kubernetes 1.21.
+
+### IPv4/IPv6 dual-stack support
+IP addresses are a consumable resource that cluster operators and administrators need to make sure are not exhausted. Having dual-stack support enables native IPv6 routing to pods and services, and it also improves a possible scaling limitation for workloads.
+
+Dual-stack support in Kubernetes means pods, services, and nodes can get IPv4 addresses and IPv6 addresses. In Kubernetes 1.21 dual-stack support is graduating from [alpha](/docs/concepts/services-networking/dual-stack/) to beta and is now enabled by default.
+
+### Graceful Node Shutdown
+[Graceful Node shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown) is graduating to beta with this release (and will now be available to a much larger group of users!). This is a hugely beneficial feature that allows Kubelet to be aware of Node shutdown, and gracefully terminate pods that are scheduled to that Node.
+
+Currently, when a node shuts down, pods do not follow the expected termination lifecycle and are not shutdown gracefully. This can introduce problems with a lot of different workloads. Going forward, Kubelet will be able to propagate the shutdown signal to pods ensuring they can terminate as gracefully as possible.
```

Comment:
I'd write:
```suggestion
Currently, when a node shuts down, pods do not follow the expected termination lifecycle and are not shut down gracefully. This can introduce problems with a lot of different workloads. Going forward, the kubelet will be able to detect imminent system shutdown through systemd, then inform running pods so they can terminate as gracefully as possible.
```

Reply:
Has been reworded per suggestion in the latest commit, thank you so much! :)